schema/deploy/views/ciip_carbon_tax_calculation.sql

```diff
@@ -71,8 +71,6 @@ begin;
     from carbon_tax_variables
  );
 
-grant all on table ggircs_portal.ciip_carbon_tax_calculation to ciip_administrator, ciip_analyst, ciip_industry_user, ciip_guest;
```

Comment:
was this removed on purpose? If so, why isn't it needed?

Reply:
Hmm I didn't explicitly remove it. I pulled some files from develop, that may have removed it.